code/game/objects/items/weapons/clown_items.dm

```diff
@@ -143,13 +143,12 @@
 	item_state = "gold_horn"
 
 /obj/item/weapon/bikehorn/golden/attack()
-	..()
 	flip_mobs()
+	return ..()

```

Comment:
Someone explain to me why this has to be return ..() instead of just ..() I am lost